### PR TITLE
Grammatically updated the tech docs (complex_numbers.rst)

### PR DESCRIPTION
**Pull request by atsigkas**
Small grammatical change in complex_numbers.rst .
-You can see the changes in the screenshot below -
Capture

### CODE REVIEW

**Comment by atsigkas** [comment]
I also opened the below issue due to understand if this normal ? (time taking to run tests )
#57545

File: docs/source/complex_numbers.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Complex Numbers
 ===============
 
 Complex numbers are numbers that can be expressed in the form :math:`a + bj`, where a and b are real numbers,
-and *j* is a solution of the equation :math:`x^2 = -1`. Complex numbers frequently occur in mathematics and
+and *j* called the imaginary unit,satisfies the equation :math:`j^2 = -1`. Complex numbers frequently occur in mathematics and
```
Comment (mrshenli) [inline docs/source/complex_numbers.rst:7]: missing space between "unit," and "satisfies"?

Comment (atsigkas) [reply docs/source/complex_numbers.rst:7]: So do you suggest to leave the proper space and then make a new pull request ? :D

Comment (mrshenli) [reply docs/source/complex_numbers.rst:7]: It should fixed before landing. Please update this PR.

Comment (atsigkas) [reply docs/source/complex_numbers.rst]: do I have to follow those steps I found in stackoverflow ? 
(I'm noobie and I dont want to make any mistake)

git reset --hard <commit key of the pull request>
Did my changes in code I wanted to do
git add
git commit --amend
git push -f origin <name of the remote branch of pull request>

Comment (atsigkas) [reply docs/source/complex_numbers.rst]: I think I done it, now I'm waiting again . 
As you can imagine I'm new here and trying to understand how git works and github works

**Review by mrshenli — APPROVED** [review]
Thanks for fixing!

**Comment by atsigkas** [comment]
@mrshenli  unfortunately it  didn't pass again....

**Pull request closed without merging**